plugins/Files/js/global.js

```diff
@@ -85,7 +85,7 @@ $('.dropdown .button').click(function() {
 			$('.dropdown li').hide('fast');
 			$('#paste-ascii').show('fast');
 			$('#paste-ascii input').focus();
-			return; // Don't close dropdown
+			break;
```

Comment:
whoops :I
thanks for popping in to help us debug. We have our hands full at the moment.